apps/web/app/future/settings/organizations/[id]/about/page.tsx

```diff
@@ -1,11 +1,15 @@
-import LegacyPage, { WrappedAboutOrganizationPage } from "@pages/settings/organizations/[id]/about";
 import { _generateMetadata } from "app/_utils";
 import { WithLayout } from "app/layoutHOC";
 
+import LegacyPage, { LayoutWrapper } from "~/settings/organizations/[id]/about-view";
+
 export const generateMetadata = async () =>
   await _generateMetadata(
     (t) => t("about_your_organization"),
     (t) => t("about_your_organization_description")
   );
```

Comment:
@hbjORbj what are your thoughts on using the `preparePageMetadata` on all app router pages? I'm thinking if we need to change how we generate metadata it would all be in one place.

Reply:
I agree with you, that will be cleaner. But I think it's a lower-pri compared to adding missing routes to App Router / extracting pages into `/modules`. Will create a ticket :)